extra_data/tests/test_reader_mockdata.py

```diff
@@ -1050,3 +1070,14 @@ def test_proc_legacy_sources(mock_modern_spb_proc_run):
         assert run.get_dtype(det_mod0, 'image.data') == np.float32
 
     assert run.select(det_mod0).all_sources == {det_mod0}
+
+
+def test_datacollection_contains(mock_fxe_control_data):
+    run = H5File(mock_fxe_control_data)
+
+    assert 'FXE_XAD_GEC/CAM/CAMERA:daqOutput' in run
+    assert 'MY/LITTLE/PONY' not in run
+    assert 'MY/LITTLE/PONY', 'actualPosition' not in run
```

Comment:
```suggestion
    assert ('MY/LITTLE/PONY', 'actualPosition') not in run
```

It may have been doing the right thing before, but parentheses are easier than thinking about operator precedence. :wink:

Reply:
Indeed I checked back then, but good point for a test.

Essentially the worded operators come after all sign operators: https://docs.python.org/3/reference/expressions.html#operator-precedence